apps/astarte_pairing/lib/astarte_pairing/fdo/owner_onboarding/prove_ov_hdr.ex

```diff
@@ -78,21 +84,27 @@ defmodule Astarte.Pairing.FDO.OwnerOnboarding.ProveOVHdrPayload do
     # Indicates the maximum message size the Owner Service can handle.
     # The Device uses this to fragment large responses if necessary.
     field :max_owner_message_size, non_neg_integer()
+
+    # CBOR encoded ov header
+    field :cbor_ov_header, binary()
+
+    # CBOR encoded hmac
+    field :cbor_hmac, binary()
```

Comment:
Nit: the typedoc above says `@typedoc "The 8-element payload to be signed inside COSE_Sign1."`, these fields make it 10